code/frameworks/nextjs/package.json

```diff
@@ -145,7 +145,7 @@
     }
   },
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.17.0"
```

Comment:
We cannot drop support for Node v16 with a minor release. This has to stay with v16 until we release a new major version, because we will still support Next.js 12 and 13

Reply:
Right, it also doesn't make sense to out of nowhere drop Node.js v16, major libraries are only now adding minimum requirement to Node.js v16; Even tho it is technically EOL. (Downstream != Upstream 🤷)

Reply:
@valentinpalkovic @ovflowd Thanks for the comments, totally understand this, I reverted it back to 16 🙂